cves/CVE-2017-15706.yml

```diff
@@ -132,8 +144,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: The documentation of common gateway interfaces
+  name: docs
 
 interesting_commits:
   question: |
```

Comment:
"If there are no interesting commits, demonstrate that you completed this section by explaining what happened between the VCCs and the fix."
There should be some descriptions of what was happening between commits even if none of them were particularly interesting.

Reply:
added description.